homeassistant/components/trace/utils.py

```diff
@@ -1,22 +1,26 @@
 """Helpers for script and automation tracing and debugging."""
 from collections import OrderedDict
+from typing import Any, TypeVar
 
+_KT = TypeVar("_KT")  # key type
+_VT = TypeVar("_VT")  # value type
```

Comment:
```suggestion
_KT = TypeVar("_KT")
_VT = TypeVar("_VT")
```
The comments aren't really necessary IMO. `_KT` and `_VT` are the default names for key / value generics.